### ISSUE TITLE
No borders / rounded corners in OVERVIEW mode when `dwindle:no_gaps_when_only = 1`

### ISSUE THREAD

**Issue by MahouShoujoMivutilde** [DreamMaoMao/hycov#29]
## Demo

See the
<details>
  <summary>screenshot</summary>

![23-12-2023-02_44_35](https://github.com/DreamMaoMao/hycov/assets/14999778/d14f950f-bf33-4a16-945a-c2b12646000c)

</details>

### The problem

Lower right window is the only window open on 1st workspace, which matches conditions for `no_gaps_when_only` to trigger. Notice how its corners are rectangular and there is no border _even in overview mode when there are other windows._

(The rest is on 3rd workspace, so they already have borders and gaps.)

### Expected behavior

All windows have the same borders / rounded corners.

### What I've done

I've tried to work around this by just turning `no_gaps_when_only` off before the overview.

```
bind = SUPER,grave,exec, hyprctl --batch "keyword dwindle:no_gaps_when_only 0; dispatch hycov:toggleoverview; keyword dwindle:no_gaps_when_only 1"

```

But there is a race condition and `keyword dwindle:no_gaps_when_only 1` doesn't wait till I select the window in overview, so it only sometimes fixes the issue.

Which could theoretically be worked around by listening to events and watching for workspace being renamed from OVERVIEW to something different... But at this point I think it makes more sense to just do it in the plugin.

E.g. save `no_gaps_when_only` value and turn it off when entering the overview mode, restore the value when leaving it.

### Version

hycov be28246
hyprland 0.33.1-2 (from arch repo)

**Comment by DreamMaoMao** [comment]
has fix [58cffcf](https://github.com/DreamMaoMao/hycov/commit/58cffcfc425eb67b6101c544c13d6525c3f1b94b)

**Comment by MahouShoujoMivutilde** [comment]
Thank you for the quick fix!

It does fix it, but now there are no shadows in overview mode for every window despite me having `decoration:drop_shadow = true` and no rules to contradict it.

**Comment by DreamMaoMao** [comment]
have fixed:[9eb6131](https://github.com/DreamMaoMao/hycov/commit/9eb61310005a4063b94556fbcfa7de46d59f7eab)
